Assets/LeapMotionModules/ElementRenderer/Resources/BakedRenderer.cginc

```diff
@@ -0,0 +1,218 @@
+#include "Assets/LeapMotionModules/ElementRenderer/Resources/LeapGui.cginc"
```

Comment:
Actually I'm not 100% sure.  I think so, I think that if they are not, then Unity will fail to compile the shader during export.  I will have to verify and get back to you.